data/data.json

```diff
@@ -11622,6 +11628,10 @@
         }
     },
     "strings": {
+        "4201": "Dark Age",
+        "4202": "Feudal Age",
+        "4203": "Castle Age",
+        "4204": "Imperial Age",
```

Comment:
These will have to get extracted from the localization file. I'll keep an eye on these.

Reply:
The python script has already been updated to do so on master, so that should be easy